cli/src/forge.rs

```diff
@@ -128,6 +132,72 @@ fn main() -> eyre::Result<()> {
         Subcommands::Snapshot(cmd) => {
             cmd.run()?;
         }
+        Subcommands::Fmt { path, root, check } => {
+            if path.is_some() && root.is_some() {
```

Comment:
can we solve this on the `clap` level with `conflicts_with?`

Reply:
nice one, fixed